src/components/AddTableForm/AddTableForm.test.js

```diff
@@ -0,0 +1,18 @@
+import { render, fireEvent } from "@testing-library/react";
+import AddTableForm from "./AddTableForm";
+
+describe("AddTableForm", () => {
+  it("should match the snapshot", () => {
```

Comment:
Created a  handy snapshot test to ensure any future changes should not do any un-intended changes to the component structure.